x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/layer/compatible_layer.tsx

```diff
@@ -77,14 +77,7 @@ export const CompatibleLayer: FC<Props> = ({ layer, layerIndex, embeddable }) =>
   const [state, setState] = useState<STATE>(STATE.DEFAULT);
   const [createError, setCreateError] = useState<{ text: string; errorText: string } | null>(null);
   const quickJobCreator = useMemo(
-    () =>
-      new QuickJobCreator(
-        data.dataViews,
-        uiSettings,
-        data.query.timefilter.timefilter,
-        share,
-        mlApiServices
-      ),
+    () => new QuickJobCreator(uiSettings, data.query.timefilter.timefilter, share, mlApiServices),
```

Comment:
the lens start contract will need to come from `useMlFromLensKibanaContext` to be supplied to `QuickJobCreator` here

Reply:
done

Reply:
done